src/shared/components/studyTagsTooltip/StudyTagsTooltip.tsx

```diff
@@ -154,7 +154,7 @@ export default class StudyTagsTooltip extends React.Component<
     renderTooltip() {
         return (
             <DefaultTooltip
-                key={this.props.key}
+                //key={this.props.key}
```

Comment:
what's this about?

Reply:
Hi @alisman. I saw an error in the browser console saying that key is not a prop. I checked and indeed, key is not a property of DefaultTooltip. Not sure why it is there so I only commented it. The error no longer appears and the tooltips work fine. You can see a screenshot in the PR description.